products/firefox.md

```diff
@@ -12,10 +12,11 @@ changelogTemplate: |
 sortReleasesBy: releaseDate
 releases:
 
--   releaseCycle: "101"
+-   releaseCycle: "102"
     eol: false
-    latest: "101.0"
-    releaseDate: 2022-05-31
+    latest: "102.0.0"
```

Comment:
Version <code>102.0<del>.0</del></code> does not exist. There is only `102.0`, see https://www.mozilla.org/en-US/firefox/releases/. Because of that, the auto-generated link on https://endoflife.date/firefox from this version tag is invalid:
- ~~https://www.mozilla.org/firefox/102.0.0/releasenotes/~~

It should be
- [https://www.mozilla.org/firefox/**102.0**/releasenotes/](https://www.mozilla.org/en-US/firefox/102.0/releasenotes/)